p/themes/Origine/origine.css

```diff
@@ -2,10 +2,19 @@
 
 /*=== FONTS */
 @font-face {
-	font-family: "OpenSans";
-	src: url("../fonts/openSans.woff") format("woff");
+  font-family: 'Open Sans';
+  font-style: normal;
+  font-weight: 400;
+  src: url('../fonts/opensans.eot'); /* IE9 Compat Modes */
+  src: local('Open Sans'), local('OpenSans'),
+       url('../fonts/opensans.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+       url('../fonts/opensans.woff2') format('woff2'), /* Super Modern Browsers */
+       url('../fonts/opensans.woff') format('woff'), /* Modern Browsers */
+       url('../fonts/opensans.ttf') format('truetype'), /* Safari, Android, iOS */
+       url('../fonts/opensans.svg#OpenSans') format('svg'); /* Legacy iOS */
```

Comment:
As far as I can see, the market share of iOS < 4.3 is below 0.1%. And it will still work even if the font is different. So I suggest dropping this line too.